packages/block-library/src/image/edit.native.js

```diff
@@ -631,5 +691,12 @@ export default compose( [
 			),
 		};
 	} ),
+	withDispatch( ( dispatch ) => {
+		return {
+			closeSettingsBottomSheet() {
+				dispatch( 'core/edit-post' ).closeGeneralSidebar();
```

Comment:
Ah, [like recently](https://github.com/WordPress/gutenberg/pull/30806#discussion_r613207635), let's not use store name literals directly, thanks!

Reply:
Ah, thank you! I'll refactor this in the new PR I'm creating for the issue. :)